src/transformState.js

```diff
@@ -6,6 +6,34 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
 for (const action of actions)  {
```  
imho this will be better

Reply:
yes, thank you!